runner_service/services/playbook.py

```diff
@@ -214,8 +215,17 @@ def start_playbook(playbook_name, vars=None, filter=None, tags=None):
 
     # this should just be run_async, using 'run' hangs the root logger output
     # even when backgrounded
+
+    artifacts_dir = os.path.join(configuration.settings.playbooks_root_dir, "artifacts")
+    if configuration.settings.max_artifacts is not None:
+        remove_oldest_artifacts(artifacts_dir)
```

Comment:
I think we should define a standalone cron like job to remove old artifacts and not removing them on each playbook execution

Reply:
When using cron we would not have constant number of artifacts and the effect would be same,
Unless we would want to define the maximum age of the artifact, but that would require larger changes.

Reply:
Yes, I think we should introduce configuration option in config.yaml to define maximum age of artifacts (by default for example 30 days) and customers with high load could change the default value as needed.

So having a job which is started during ansible-runner-service startup can perform cleanup and then schedule it's own next execution within maximum age interval is the solution I'd prefer.

Reply:
Okay, this will be a larger change. I will revert changes here and create separate PR and continue there.